ee/tabby-db/src/users.rs

```diff
@@ -206,6 +206,13 @@ impl DbConn {
         .await?;
         Ok(())
     }
+
```

Comment:
```suggestion
// FIXME(boxbeam): consider implement this with a in-memory caching in future.
```

Reply:
But I did implement it with in-memory caching? Not sure what is expected here. The caching just isn't done at the db level.